docs/GlobalAPI.md

```diff
@@ -994,6 +988,8 @@ For simplicity `test.each` is picked for the examples, but the type inference is
 The array of objects API is most verbose, but it makes the type inference a painless task. A `table` can be inlined:
 
 ```ts
+import {test} from '@jest/globals';
```

Comment:
Similar to MockFunctionApi docs – note at the top and explicit imports in TypeScript Usage sections.